osc-server/pom.xml

```diff
@@ -165,13 +165,6 @@
             <version>${slf4j.version}</version>
             <scope>provided</scope>
         </dependency>
-         <dependency>
```

Comment:
It is strange that this is not needed here but it is needed in the control project. Should it be the other way around?

Reply:
I actually observed it to fail without these dependencies. But with my recent change these dependencies are not needed.